dev-tools/reconfigurator-cli/tests/output/target-release-stdout

```diff
@@ -560,11 +559,11 @@ to:   blueprint af934083-59b5-4bf6-8966-6fb5292c29e1
-    ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
--   sled      0      model0        serial0         7e6667e646ad001b54c8365a3d309c03f89c59102723d38d01697ee8079fe670   1.0.0              Sp { expected_active_version: ArtifactVersion("0.0.1"), expected_inactive_version: NoValidVersion }
-+   sled      1      model1        serial1         7e6667e646ad001b54c8365a3d309c03f89c59102723d38d01697ee8079fe670   1.0.0              Sp { expected_active_version: ArtifactVersion("0.0.1"), expected_inactive_version: NoValidVersion }
+    ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    sp_type   slot   part_number   serial_number   artifact_hash                                                        artifact_version   details                                                                                                                                                                                                                                                                             
+    ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+*   sled      0      model0        serial0         - 04e4a7fdb84acca92c8fd3235e26d64ea61bef8a5f98202589fd346989c5720a   1.0.0              - Rot { expected_active_slot: ExpectedActiveRotSlot { slot: A, version: ArtifactVersion("0.0.2") }, expected_inactive_version: NoValidVersion, expected_persistent_boot_preference: A, expected_pending_persistent_boot_preference: None, expected_transient_boot_preference: None }
+     └─                                            + 7e6667e646ad001b54c8365a3d309c03f89c59102723d38d01697ee8079fe670                      + Sp { expected_active_version: ArtifactVersion("0.0.1"), expected_inactive_version: NoValidVersion }                                                                                                                                                                               
```

Comment:
The artifact version isn't showing for the pending SP update, not sure what's going on here 🤔 On line 537 you can definitely see `artifact_version` is set to `1.0.0`. Probably a bug with a `writeln!` somewhere?

Reply:
Ok, looking at the code this seems like it's expected. It's not showing the slot, part_number or serial_number either because they are they are the same (i.e. no changes from the previous one)